chat/ConversationSettings.vue

```diff
@@ -101,6 +101,26 @@
         </option>
       </select>
     </div>
+    <div class="mb-3">
+      <label class="control-label" :for="'enableLogging' + conversation.key">{{
+        l('conversationSettings.enableLogging')
+      }}</label>
+      <select
+        class="form-select"
+        :id="'enableLogging' + conversation.key"
+        v-model="enableLogging"
+      >
+        <option :value="setting.Default">
```

Comment:
This isn't a comment about your work, but I just want to say that when we do global settings that this shit should really just be a generic component instead of something we copy paste a bajillion times.